src/SwedbankPay.Sdk.Tests/TestHelpers/MemoryFileInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.IO;
+using Microsoft.Extensions.FileProviders;
+
+namespace SwedbankPay.Sdk.Tests.TestHelpers
+{
+    public class MemoryFileInfo : IFileInfo
```

Comment:
What is this class used for?

Reply:
I noticed it was there earlier also. But i couldn't see that either this or MockFileProvider is used.

Reply:
Good catch. It seems this file is no longer in use.

Reply:
If `MemoryFileInfo` is not in use, can't it be deleted?

Reply:
It is used in TestHelper. If build in Release.